components/match2/commons/match_group_input.lua

```diff
@@ -589,17 +596,17 @@ function MatchGroupInput.readOpponent(match, opponentIndex, options)
 			options,
 			{timestamp = match.timestamp, timezoneOffset = match.timezoneOffset}
 		)
-	end
 
-	if options.pagifyPlayerNames then
-		Array.forEach(opponent.players or {}, function(player)
-			player.pageName = Page.pageifyLink(player.pageName)
-		end)
+		if options.pagifyPlayerNames then
+			Array.forEach(opponent.players or {}, function(player)
+				player.pageName = Page.pageifyLink(player.pageName)
+			end)
+		end
 	end
 
 	local record = MatchGroupInput.mergeRecordWithOpponent(opponentInput, opponent, substitutions)
 
-	if options.pagifyOpponentName then
+	if options.pagifyOpponentName and opponent.type == Opponent.team then
```

Comment:
Maybe this config should be renamed `pagifyTeamNames`?

Reply:
imo oos as it would need changing in most customs

Reply:
Only the 10 migrated no?